Tests/DatadogTests/Datadog/RUM/RUMEvent/RUMEventBuilderTests.swift

```diff
@@ -18,8 +18,8 @@ class RUMEventBuilderTests: XCTestCase {
         )
 
         XCTAssertEqual(event.model.attribute, "foo")
-        XCTAssertEqual((event.attributes as? [String: String])?["foo"], "bar")
-        XCTAssertEqual((event.attributes as? [String: String])?["fizz"], "buzz")
+        XCTAssertEqual((event.model.context?.contextInfo as? [String: String])?["foo"], "bar")
+        XCTAssertEqual((event.model.context?.contextInfo as? [String: String])?["fizz"], "buzz")
     }
```

Comment:
I'm surprised we don't test if `RUMEventBuilder` sets the user attributes here. We could do it easily by mocking the value in the provider:
```diff
- userInfoProvider: .mockAny(),
+ userInfoProvider: .mockWith(
+   userInfo: .init(id: nil, name: nil, email: nil, extraInfo: ["bazz": "buzz"])
+ ),
```
WDYT?

Reply:
By applying [this change](https://github.com/DataDog/dd-sdk-ios/pull/586/files#r702641511), we won't be setting the `userInfo` in here.
This an be covered by [this test](https://github.com/DataDog/dd-sdk-ios/blob/02963df9beca1734bbabf3bd6730281c00d5664b/Tests/DatadogTests/Datadog/RUM/RUMEvent/RUMUserInfoProviderTests.swift#L34) tho!